deny.toml

```diff
@@ -1,61 +1,62 @@
 [sources.allow-org]
```

Comment:
I think `cargo-deny` is no longer used, at least not in the CI. Maybe we should delete this file entirely?